package-lock.json

```diff
@@ -12,7 +12,7 @@
         "@fortawesome/free-brands-svg-icons": "^6.0.0",
```

Comment:
No need to push this lock file, they can install by running `npm install` or `yarn install` for themself.